test/runtests.jl

```diff
@@ -15,8 +15,11 @@ test_infile = @__FILE__
 test_compressed = joinpath(tmp, "runtests.jl.gz")
 test_empty = joinpath(tmp, "empty.jl.gz")
 
-@windows_only gunzip="gunzip.exe"
-@unix_only    gunzip="gunzip"
+if is_windows()
+    gunzip = "gunzip.exe"
+elseif is_unix()
+    gunzip = "gunzip"
+end
```

Comment:
I just merged this, but this should probably be made exhaustive--either by removing the second test, or by having a third else which throws an error (i.e., for the potential case of Julia being ported to some non-unix, non-windows system).